### PR TITLE
Package ppx_system.1.0

### PR DESCRIPTION
**Pull request by ArtichOwO**
### `ppx_system.1.0`
A ppx to know host operating system at compile time

---
* Homepage: https://github.com/Psi-Prod/ppx_system
* Source repo: git+https://github.com/Psi-Prod/ppx_system.git
* Bug tracker: https://github.com/Psi-Prod/ppx_system/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ppx_system/ppx_system.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+synopsis: "A ppx to know host operating system at compile time"
+maintainer: ["tim.arnouts@protonmail.com"]
+authors: ["The ppx_system programmers"]
+license: "GPL-3.0"
+homepage: "https://github.com/Psi-Prod/ppx_system"
+bug-reports: "https://github.com/Psi-Prod/ppx_system/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.3"}
+  "ppxlib"
```
Comment (mseri) [inline packages/ppx_system/ppx_system.1.0/opam:12]:
```suggestion
  "ppxlib" {>= "0.18.0"}
```
due to
```
# File "lib/expander.ml", line 25, characters 40-65:
# Error: The constructor Pconst_string expects 2 argument(s),
#        but is applied here to 3 argument(s)
```

**Comment by kit-ty-kate** [comment]
The tests fail on macOS:
```
#=== ERROR while compiling ppx_system.1.0 =====================================#
# context              2.1.2 | macos/x86_64 | ocaml.4.14.0 | pinned(https://github.com/Psi-Prod/ppx_system/archive/refs/tags/1.0.tar.gz)
# path                 ~/.opam/default/.opam-switch/build/ppx_system.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ppx_system -j 11 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/ppx_system-60909-4b7f61.env
# output-file          ~/.opam/log/ppx_system-60909-4b7f61.out
### output ###
# File "test/dune", line 2, characters 7-22:
# 2 |  (name test_ppx_system)
#            ^^^^^^^^^^^^^^^
# (cd _build/default/test && ./test_ppx_system.exe)
# Fatal error: exception Assert_failure("test/test_ppx_system.ml", 3, 0)
```

**Comment by ArtichOwO** [comment]
Hi! I'm very sorry, I completely forgot about this @~@
@Tim-ats-d did rework the tests so it works on macOS too, but how am I supposed to update the PR?
redo an `opam publish`?

**Comment by mseri** [comment]
Yes, if you re-tag and run opam-publish it should force push and update this PR automatically.

**Comment by ArtichOwO** [comment]
thanks! will do

**Comment by mseri** [comment]
Looks good, thanks. Consider announcing the release on discuss.ocaml.org. We have a community category and an announcement tag for this purpose

**Pull request merged**